### ISSUE TITLE
Element moves around erratically when using mouse to adjust position of various elements (stems/augmentation dots etc.)

### ISSUE THREAD

**Issue by wizofaus** [musescore/MuseScore#25009]
### Issue type

UX/Interaction bug (incorrect behaviour)

### Description with steps to reproduce

1. Create or open a new score with a dotted 1/4 note
2. Select just the augmentation dot, enter edit mode (F2), then use the mouse to drag it around
3. Expected: dot should follow mouse. Actual: dot moves around erratically.
4. Exit edit mode for the dot, then select just the stem of the note (release mouse afterwards)
5. Click and drag the mouse from somewhere in the middle of the stem (not where the drag handle is)
6. Expected: stem should lengthen/shorten based on mouse movements. Actual: stem lengthens/shortens erratically and uncontrollably

Also after such an operation, undo appears to undo not just the position/length adjustment but creating the note in the first place.
 

### Supporting files, videos and screenshots

https://github.com/user-attachments/assets/a486aed7-f2dc-49cb-a61b-1403a2132ceb

### What is the latest version of MuseScore Studio where this issue is present?

4.3.1

### Regression

I was unable to check

### Operating system

Windows 11

### Additional context

Possibly related to other similar issues where dragging with mouse produces unexpected results

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by bkunda** [comment]
I'd move that these click-drag behaviours simply be disabled. 

Especially for adjusting stem lengths. It doesn't really make sense to change the length of a stem by clicking somewhere random in the middle of the stem (and not by using the much more obvious drag handle at the end of the stem).

Certainly the erratic behaviour is buggy, but I don't think we should fix it by giving it additional functionality.

**Comment by MarcSabatella** [comment]
I agree wityh respect to stem - that doesn't make make sense to support.  To dot behavior is just a bug, though.  Dragging the dot should work but does not.

**Comment by wizofaus** [comment]
No real issue with disabling drag for stem length if you start dragging somewhere other than the drag handle, though it happened to me when I'd intended to use drag handle, but must have just missed slightly.
